api.ts

```diff
@@ -199,11 +204,16 @@ export interface WorkspaceBrowseConfiguration {
     readonly compilerPath?: string;
 
     /**
-     * The C or C++ standard to emulate. This field defaults to "c++17" and will only be used if
+     * The C standard to emulate. This field defaults to "c++17" and will only be used if
      * [compilerPath](#WorkspaceBrowseConfiguration.compilerPath) is set.
      */
-    readonly standard?: "c89" | "c99" | "c11" | "c++98" | "c++03" | "c++11" | "c++14" | "c++17";

```

Comment:
For browsing, all we use the compilerPath for is to query the system includes so we know which paths to tag parse.  Therefore, the purpose of this setting at this time is really just to determine whether we should add the C++ include paths or not.

Reply:
Well,the reason i want to add the cStandard/cppStandard is because the [vscode-cmake-tools](https://github.com/vector-of-bool/vscode-cmake-tools) always return [`c++17`](https://github.com/vector-of-bool/vscode-cmake-tools/blob/5f7bca659901ce2ecd0eab27302f02e3208e0d98/src/cpptools.ts#L38) by default. 
Then,
![screenshot from 2019-01-23 01-15-07](https://user-images.githubusercontent.com/11146248/51552807-5f160e80-1eac-11e9-850a-580a53434368.png).
It won't affect the building,but you know....

Reply:
It sounds like this is a bug in the CMake Tools extension then?

Reply:
But it's just returning what interface `WorkspaceBrowseConfiguration` ask. [link](https://github.com/vector-of-bool/vscode-cmake-tools/blob/5f7bca659901ce2ecd0eab27302f02e3208e0d98/src/cpptools.ts#L213)

Reply:
I do misunderstanding the interface `SourceFileConfigurationItem` though

Reply:
After setting the value,  I think updating  the cmake-tool will fix the problem. All thanks

Reply:
It looks like CMake Tools may incorrectly set c++17 for C files if `-std` is not on the compilation command line.  The default should probably be set based on the extension of the file (e.g. c11 for .c files, and c++17 for the others)

Reply:
Yes! I'm changing my origin issue in cmake-tool. Thank you for your help 👍!